databases/dbeaver-community/Portfile

```diff
@@ -1,44 +1,48 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem                  1.0
-PortGroup                   github 1.0
+PortSystem          1.0
+PortSystem          1.0
```

Comment:
no need to duplicate this line (also, as a nitpick, the commit doen’t really “fix” the style as there wasn’t anything wrong with it before).